docs/handler-api.mdx

```diff
@@ -1075,31 +1075,54 @@ NemirtingasEpicEmu.json edition from a handler example:
 ---
 ### 11. Nemirtingas Galaxy GoG Emulator
 
-Automatically set up Nemirtinga's Galaxy Emulator in Nucleus.
+Nemirtinga's Galaxy Emulator works by substituting the game's Galaxy64.dll file with its own custom dll.
+
+First, set up Nemirtinga's Galaxy Emulator in Nucleus.
+
+`Game.UseNemirtingasGalaxyEmu = true;   //Set to true to emulate GOG Galaxy`
```

Comment:
Replace this with the code below, that way you'll get syntax highlighting like this

```js
Game.UseNemirtingasGalaxyEmu = true;   //Set to true to emulate GOG Galaxy
```
> \`\`\`js
Game.UseNemirtingasGalaxyEmu = true;   //Set to true to emulate GOG Galaxy
\`\`\`